docs: Locating resources deployed by nitric (#654)
nitrictech/docs: docs/providers/pulumi/aws/index.mdx

  for full details on credentials and configuration.
</Note>

> ## Locating resources deployed with Nitric

Reviewer: ```suggestion
## Locating deployed resources
```

> Nitric creates a resource tag manager group and tags all possible resources to be referenced by this group. You can locate resources in the console by going to the [AWS Console](https://console.aws.amazon.com/)

Reviewer: ```suggestion
This Nitric AWS provider creates a resource tag manager group and tags all possible resources to be referenced by this group. You can locate resources using the [AWS Console](https://console.aws.amazon.com/).
```

> Navigate to the resource groups service:

Reviewer: ```suggestion
Start by navigating to the [Resource Groups service](https://console.aws.amazon.com/resource-groups/home):
```

![resource group search image](/docs/images/docs/aws-rg.png)

> Find your resource group in the resource groups list and click it:
> 
> ![resource group list image](/docs/images/docs/aws-rg-screen.png)
> 
> Resource groups are conventionally named `<project-name>-<stack-name>`
> 
> So in the example above, the project name is `api-testing` and the stack name is `aws`.

Reviewer: ```suggestion
Find your stack's resource group in the list and click it:

<Note>
  Resource groups are conventionally named `<project-name>-<stack-name>`

  In this example the project name is `api-testing` and the stack name is `aws`.
</Note>

![resource group list image](/docs/images/docs/aws-rg-screen.png)
```

> More recent versions of the Nitric AWS provider also output a direct link to it's resource group in the console from running `nitric up`

Reviewer: ```suggestion
More recent versions of the Nitric AWS provider also display a direct link to the resource group in output from `nitric up`.
```
